README.md

```diff
@@ -110,6 +110,11 @@ See the build [guide](BUILD.md).
 
 Please see our [guide for contributing to cuML](CONTRIBUTING.md).
 
+## References
+
+For additional details on the technologies behind cuML, as well as a broader overview of the Python Machine Learning landscape, see [_Machine Learning in Python: Main developments and technology trends in data science, machine learning, and artificial intelligence_ (2020)](https://arxiv.org/abs/2002.04803) by Sebastian Raschka, Joshua Patterson, and Corey Nolet.
```

Comment:
What do you think about also including the bibtex for citing cuML in scientific research? I know that was one of the big drivers behind wanting to include this in the README / docs. 

For example: https://ogrisel.github.io/scikit-learn.org/sklearn-tutorial/about.html#citing-scikit-learn

Reply:
Here's the bibtex

```
@article{raschka2020machine,
  title={Machine Learning in Python: Main developments and technology trends in data science, machine learning, and artificial intelligence},
  author={Raschka, Sebastian and Patterson, Joshua and Nolet, Corey},
  journal={arXiv preprint arXiv:2002.04803},
  year={2020}
}
```